From e902e7cdf57ccc677442d9cc86a907db78e6daab Mon Sep 17 00:00:00 2001
From: jschyra <jschyra@90b65887-3827-0410-9a23-83215b262276>
Date: Fri, 16 Dec 2011 14:18:17 +0000
Subject: [PATCH 0228/1077] BUGFIX:  - STARTED durch STARTUP ersetzt. Beim
 hochfahren wurde vorher ein falscher Status an den Server gesendet.

git-svn-id: svn://localhost/SelfServiceCommon/trunk@278 90b65887-3827-0410-9a23-83215b262276
---
 .../Massai/cpp/LogService/HOBBIT/bbWin/Kiosk.xslt           | 6 +++---
 .../Massai/cpp/LogService/HOBBIT/bbWin/Paper.xslt           | 5 +----
 2 files changed, 4 insertions(+), 7 deletions(-)

diff --git a/SelfServiceCommon/Massai/cpp/LogService/HOBBIT/bbWin/Kiosk.xslt b/SelfServiceCommon/Massai/cpp/LogService/HOBBIT/bbWin/Kiosk.xslt
index 321f0759..f828e3db 100644
--- a/SelfServiceCommon/Massai/cpp/LogService/HOBBIT/bbWin/Kiosk.xslt
+++ b/SelfServiceCommon/Massai/cpp/LogService/HOBBIT/bbWin/Kiosk.xslt
@@ -14,7 +14,7 @@
             <xsl:when test="@st = 'UNDER_SERVICE'">
                 <xsl:text>yellow</xsl:text>
             </xsl:when>
-            <xsl:when test="@st = 'STARTED'">
+            <xsl:when test="@st = 'STARTUP'">
                 <xsl:text>yellow</xsl:text>
             </xsl:when>
             <xsl:otherwise>
@@ -42,7 +42,7 @@
                     <xsl:when test="@st = 'ACTIVE'">
                         <xsl:text>&amp;yellow</xsl:text>
                     </xsl:when>
-                    <xsl:when test="@st = 'STARTED'">
+                    <xsl:when test="@st = 'STARTUP'">
                         <xsl:text>&amp;yellow</xsl:text>
                     </xsl:when>
                     <xsl:otherwise>
@@ -76,7 +76,7 @@
                     <xsl:when test="@st = 'OPEN'">
                         <xsl:text>&amp;yellow</xsl:text>
                     </xsl:when>
-                    <xsl:when test="@st = 'STARTED'">
+                    <xsl:when test="@st = 'STARTUP'">
                         <xsl:text>&amp;yellow</xsl:text>
                     </xsl:when>
                     <xsl:otherwise>
diff --git a/SelfServiceCommon/Massai/cpp/LogService/HOBBIT/bbWin/Paper.xslt b/SelfServiceCommon/Massai/cpp/LogService/HOBBIT/bbWin/Paper.xslt
index b0d4ad0e..82048407 100644
--- a/SelfServiceCommon/Massai/cpp/LogService/HOBBIT/bbWin/Paper.xslt
+++ b/SelfServiceCommon/Massai/cpp/LogService/HOBBIT/bbWin/Paper.xslt
@@ -24,7 +24,7 @@
                 <xsl:when test="@st = 'LOW'">
                     <xsl:text>yellow</xsl:text>
                 </xsl:when>
-                <xsl:when test="@st = 'STARTED'">
+                <xsl:when test="@st = 'STARTUP'">
                     <xsl:text>yellow</xsl:text>
                 </xsl:when>
                 <xsl:when test="@st = 'SHUTDOWN'">
@@ -33,9 +33,6 @@
                 <xsl:when test="@st = 'EMPTY'">
                     <xsl:text>red</xsl:text>
                 </xsl:when>
-                <xsl:when test="@st = 'STARTED'">
-                    <xsl:text>yellow</xsl:text>
-                </xsl:when>
                 <xsl:otherwise>
                     <xsl:text>red</xsl:text>
                 </xsl:otherwise>
-- 
2.41.0.windows.1

